### ISSUE TITLE
[HOLD for payment 2023-05-04] [$2000] No background color on hover on copy to clipboard in workspace->reimburse expenses

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#16574]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app
2. Open workspaces
3. Open any workspace
4. Open reimburse expenses
5. Hover on copy to clipboard icon
6. Get back to home page and open any report
7. Click on that report user profile to open user details page for that user
8. Hover on copy to clipboard icon

## Expected Result:
Copy to clipboard in reimburse expenses should have background color effect on hover and properly spaced from the text.

## Actual Result:
Inconsistent behavior for same copy to clipboard icon on hover between user details and reimburse expenses page

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.90-4
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/228032891-b3b8c8d5-a029-4f4c-b151-4363c7eac2c7.mp4

https://user-images.githubusercontent.com/43996225/228032916-d571ba5e-0899-4320-b23d-361f9ecbd685.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1679905653532999

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0125054b3431e347fc</li>
        <li>Upwork Job ID: 1643686746350624768</li>
        <li>Last Price Increase: 2023-04-19</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
Triggered auto assignment to @mallenexpensify (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by mallenexpensify** [comment]
@dhanashree-sawant, before moving this along, can you do an audit to see if there are any other instances where there are inconsistencies with the way the copy button looks in the app? I'd prefer to fix multiple at once if possible vs creating multiple issues for the same inconsistency)

**Comment by dhanashree-sawant** [comment]
Hi @mallenexpensify , Sure will check throughout the app and let you know.

**Comment by dhanashree-sawant** [comment]
Hi @mallenexpensify, I have gone through all the pages and we only use copy to clipboard on message hover, user details and reimburse expenses. Only reimburse expenses page has the inconsistency which we will need to resolve.

**Comment by dhanashree-sawant** [comment]
Hi @mallenexpensify, recently we have again added new contact methods page in profile, even on that page we will need to resolve copy to clipboard icon issue

**Comment by MelvinBot** [comment]
@mallenexpensify 6 days overdue. This is scarier than being forced to listen to Vogon poetry!

**Comment by mallenexpensify** [comment]
Thanks @dhanashree-sawant , sorry for the delay, been sick the past week.  Moving along.

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~0125054b3431e347fc

**Comment by MelvinBot** [comment]
Current assignee @mallenexpensify is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @aimane-chnaif (`External`)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @robertjchen (`External`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by alitoshmatov** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Inconsistency in copy to clipboard buttons
### What is the root cause of that problem?
This is caused due to different approaches in coding this buttons. The correct one, `CommunicationsLink` component is using `ContextMenuItem` like this
https://github.com/Expensify/App/blob/0d2d52eaa4f5ce91585be007645ad1d4e6a230df/src/components/CommunicationsLink.js#L40L47

Whereas in `CopyTextToClipboard` it is used like this: 
https://github.com/Expensify/App/blob/0d2d52eaa4f5ce91585be007645ad1d4e6a230df/src/components/CopyTextToClipboard.js#L62L70

### What changes do you think we should make in order to solve the problem?
We should update `CopyTextToClipboard` to use the same logic as `CommunicationsLink`

### What alternative solutions did you explore? (Optional)

**Comment by alitoshmatov** [comment]
Updated code reference

**Comment by soumyajit4419** [comment]
# Proposal

# Please re-state the problem that we are trying to solve in this issue.
The copy to clipboard button is not hovered.

# What is the root cause of that problem?
The root cause of the issue is that there is no hover functionality present in `copyToClipboard` component.

# What changes do you think we should make in order to solve the problem?
We can wrap the icon present in the `copyToClipboard` component with a view and track the mouseEvent to maintain the hover state.
We can replace this [Block](https://github.com/allroundexperts/Expensify/blob/e56b9c76e8d2f058625c9dc729d700e12130c7f8/src/components/CopyTextToClipboard.js#L63-L69) with below function.

```
    <View
        style={[
            Platform.OS === 'web' && styles.copyToClipBoardButton,
            styles.alignItemsCenter, styles.justifyContentCenter,
            this.state.isHovered ? styles.hoveredButton : styles.bgTransparent,
        ]}
        onMouseEnter={
            () => {
                this.setState({isHovered: true});
            }
          }
        onMouseLeave={
            () => {
                this.setState({isHovered: false});
            }
          }
    >
        <Icon
            src={this.state.showCheckmark ? Expensicons.Checkmark : Expensicons.Clipboard}
            fill={this.state.showCheckmark ? themeColors.iconHovered : themeColors.icon}
            width={variables.iconSizeSmall}
            height={variables.iconSizeSmall}
            inline
        />
    </View>

```


# Result


https://user-images.githubusercontent.com/46092576/230205878-94772800-23c6-4a6f-a4f6-10a547a3390a.mov

https://user-images.githubusercontent.com/46092576/230205635-24b8e662-4926-41cc-8504-913d7434e02c.mov

https://user-images.githubusercontent.com/46092576/230205773-0cd6a3f6-58ae-40fe-b82f-020fa483fa81.mov




# What alternative solutions did you explore? (Optional)
NA

**Comment by robertjchen** [comment]
Thanks for the proposals and detailed screen recordings on all the platforms! 🙇 

At the moment, I'm partial to @allroundexperts 's proposal https://github.com/Expensify/App/issues/16574#issuecomment-1498029836 since it doesn't involve any platform specific logic as in the second proposal. What do you think @aimane-chnaif ? 🙏

**Comment by robertjchen** [comment]
>  we don't need to show circle background in wrapper when hovered, right? 

Ah, I was thinking of that as well. To maintain consistency, I'd say it should also show a background when hovered 👍 
<img width="306" alt="Screenshot 2023-04-09 at 5 37 40 PM" src="https://user-images.githubusercontent.com/2438855/230785120-9d8c75d4-7413-449f-a1db-fa40b0c5d277.png">

**Comment by MelvinBot** [comment]
@robertjchen @mallenexpensify @aimane-chnaif this issue was created 2 weeks ago. Are we close to approving a proposal? If not, what's blocking us from getting this issue assigned? Don't hesitate to create a thread in #expensify-open-source to align faster in real time. Thanks!

**Comment by MelvinBot** [comment]
@robertjchen @mallenexpensify @aimane-chnaif this issue was created 2 weeks ago. Are we close to approving a proposal? If not, what's blocking us from getting this issue assigned? Don't hesitate to create a thread in #expensify-open-source to align faster in real time. Thanks!

**Comment by MelvinBot** [comment]
Upwork job price has been updated to $2000

**Comment by mallenexpensify** [comment]
Bumped to $2000 to get more proposals, shared in #expensify-bugs too

**Comment by MahmudjonToraqulov** [comment]
### Proposal
### Please re-state the problem that we are trying to solve in this issue.

No background color on hover on copy to clipboard in workspace

### What is the root cause of that problem?

The styles existing in others hasn't been added to it. 

### What changes do you think we should make in order to solve the problem?
Let me give you an example in pure css and with images.

I added these odes to this div:
![image](https://user-images.githubusercontent.com/91727683/231664190-767257d9-8040-47b8-b778-5df1146108e1.png)

and these code as external style:
```
.eerr:hover{
    background: rgb(44,103,88);
}
.eerr:hover path{
    fill: white;
}
```
Note: I haven't checked others proposal for this GH issue, so, sorry if the proposal like this was alredy given.


### What alternative solutions did you explore? (Optional)
Not / yet

### Result
[screen-capture (1).webm](https://user-images.githubusercontent.com/91727683/231664870-148e34d9-5e3e-4dd6-98a7-ee6f07670d13.webm)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @shawnborton (`Design`), see [these Stack Overflow questions](https://stackoverflow.com/c/expensify/search?q=%5Bchore%5D%5Bteam-design%5D+is%3Aquestion&searchOn=1) for more details.

**Comment by mallenexpensify** [comment]
Added the design label, @shawnborton can you take a look? Specifically 
https://github.com/Expensify/App/issues/16574#issuecomment-1501180838

Thx

@MahmudjonToraqulov , after the above has been reviewed, @aimane-chnaif will address your proposal.

**Comment by MahmudjonToraqulov** [comment]
@mallenexpensify

> Added the design label, @shawnborton can you take a look? Specifically [#16574 (comment)](https://github.com/Expensify/App/issues/16574#issuecomment-1501180838)
> 
> Thx
> 
> @MahmudjonToraqulov , after the above has been reviewed, @aimane-chnaif will address your proposal.

Look forwad to hearing from you,Thnx.

**Comment by MahmudjonToraqulov** [comment]
Hey @mallenexpensify. Any updates?

**Comment by ericsonrd** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
* There’s no background color when hovering on “Copy to clipboard” icon in Workspaces > Reimburse page.
* (Additional) Background circle color could introduce inappropriate spacing before and after the icon when not hovered.
* (Additional) “Copy to clipboard” icon is too close to email address.

### What is the root cause of that problem?
* There’s only an icon button displayed without any hovering logic.
* On the additional issue of closeness to email address, there’s no spacing around the icon to separate it from email address.

### What changes do you think we should make in order to solve the problem?
* **For hovering logic issue:** A simple function to select hovering/pressing color on `CopyTextToClipboard` should be defined, then this hovering/pressing logic should be applied on “copy to clipboard” icon via Pressable Component. Since it would be wrapped by the `CopyTextToClipboard` component and bound to its spacing styles, it woldn’t break style of Inline.
* **For background circle issue:** Logic should be applied only on Copy icon, without adding a circle background to avoid inappropriate spacing when not hovered.
* **For closeness to email address issue:** A style attribute adding extra spacing should be added to the `IconWrapperStyles` inside the `Icon` component.
    
    This solutions would affect:
    
    - Workspaces > Reimburse page
    - Profile > Contact Methods page
    
**Solution example:**
    
```jsx
// CopyToClipboard.js //
    
hoveredIcon(hovered, pressed) {
    return (hovered || pressed) ? themeColors.iconHovered : themeColors.icon
}
    
<Pressable
    onPress={this.copyToClipboard}
>
    {({hovered, pressed}) => (
        <Icon
            src={this.state.showCheckmark ? Expensicons.Checkmark : Expensicons.Copy}
            fill={this.state.showCheckmark ? themeColors.iconSuccessFill : this.hoveredIcon(hovered, pressed)}
            width={variables.iconSizeSmall}
            height={variables.iconSizeSmall}
            inline
        />
    )}
</Pressable>
    
// IconWrapperStyles/index.js //
export default {top: 2, marginLeft: 1};
    
// IconWrapperStyles/index.ios.js //
export default {top: 1, marginLeft: 1};
```
    

**Additional changes:** To keep consistency across the app, a good option would be to modify the Copy icon of the Contact Details page as well, by removing the background circle. This sets a format for “email + copy icon” and leave the background circle for other icons like emojis which contain no text around:

```jsx
// BaseMiniContextMenuItem.js //

<Pressable
    ref={props.innerRef}
    focusable
    onPress={props.onPress}
    accessibilityLabel={props.tooltipText}
// -------- Remove this style prop below -------- //
    style={ 
        ({hovered, pressed}) => [
            styles.reportActionContextMenuMiniButton,
            StyleUtils.getButtonBackgroundColorStyle(getButtonState(hovered, pressed, props.isDelayButtonStateComplete)),
        ]
    }
>
    {pressableState => (
        <View style={[StyleUtils.getWidthAndHeightStyle(variables.iconSizeNormal), styles.alignItemsCenter, styles.justifyContentCenter]}>
            {_.isFunction(props.children) ? props.children(pressableState) : props.children}
        </View>
    )}
</Pressable>
```

**Results:**


https://user-images.githubusercontent.com/95394815/232339845-c0ed6b77-2d91-404a-ab51-2b3c08c75be3.mp4




https://user-images.githubusercontent.com/95394815/232339092-5a2ad68d-e010-467e-b948-0eed39ca0680.mov





- **What alternative solutions did you explore?**

N/A

**Comment by shawnborton** [comment]
I agree we should use that new copy icon (and we should probably replace that everywhere the old clipboard icon was used), and I think we should have 4px of margin after the email and before the icon.

**Comment by shawnborton** [comment]
I don't think it should have hover background color but it could change from color.icon to color.textSupporting on hover.

**Comment by shawnborton** [comment]
At the same time, none of our other icons change color like this so I'd be fine if it doesn't have a change in hover color, or we should consider adding hover color to the search/call/pin icons:
<img width="165" alt="image" src="https://user-images.githubusercontent.com/2319350/232344978-3665d01a-a95c-43f6-91fd-2ad55d1472e6.png"> <img width="165" alt="image" src="https://user-images.githubusercontent.com/2319350/232344981-32449223-6550-482e-8c42-267373c60abc.png">

**Comment by ericsonrd** [comment]
> I agree we should use that new copy icon (and we should probably replace that everywhere the old clipboard icon was used), and I think we should have 4px of margin after the email and before the icon.

I made the test and 4px adds a lot of spacing for this component (in terms of communicating that this icon belongs to the email address preceeding it). I made an example with 4px and 2px (which I'd propose) so you can see the difference:


![expMarginleft](https://user-images.githubusercontent.com/95394815/232346375-c015df31-99ef-4f21-a207-189c61f6fe25.jpg)


## Proposal
[Updated](https://github.com/Expensify/App/issues/16574#issuecomment-1510478678)

**Comment by ericsonrd** [comment]
> I don't think it should have hover background color but it could change from color.icon to color.textSupporting on hover.

About this, do you think it's better to use `themeColors.textSupporting` or `themeColors.iconHovered`?

Here are some examples to see the difference:

* **Using `themeColors.textSupporting`**

https://user-images.githubusercontent.com/95394815/232347927-efae0dd2-410b-4d1f-95e0-38344ae73f0e.mov

* **Using `themeColors.iconHovered`**

https://user-images.githubusercontent.com/95394815/232347960-448e5134-5164-4c6d-9452-6af5a7ac3d72.mov

**Comment by MahmudjonToraqulov** [comment]
Hey  @mallenexpensify @shawnborton @robertjchen @aimane-chnaif 
Has someone chcked my proposal?

**Comment by situchan** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
- there's no visual change when hover copy to clipboard icon
- email and icon are too close

### What is the root cause of that problem?
- this page is using [CopyTextToClipboard](https://github.com/Expensify/App/blob/main/src/components/CopyTextToClipboard.js) component and it doesn't handle hover state but icon color is static
- no margin/padding defined either in Text or in Icon

### What changes do you think we should make in order to solve the problem?
- wrap [Icon](https://github.com/Expensify/App/blob/74d5290dd95500b86f8b4f384b18b9aca87364a2/src/components/CopyTextToClipboard.js#L63) with Pressable to get `hovered`, `pressed` states from callback
- update Icon [fill](https://github.com/Expensify/App/blob/74d5290dd95500b86f8b4f384b18b9aca87364a2/src/components/CopyTextToClipboard.js#L65) prop to get color from `getIconFillColor` in `StyleUtils`. Param to pass: `getButtonState(hovered, pressed, this.state.showCheckmark)`
- Pressable `onPress` can be linked to `this.copyToClipboard`
- To fix space issue, add empty string (` `) to [text value](https://github.com/Expensify/App/blob/74d5290dd95500b86f8b4f384b18b9aca87364a2/src/components/CopyTextToClipboard.js#L61). We cannot add margin/padding because it's inside Text view and doesn't work on native.
```
`${this.props.text} `
```

**Comment by mallenexpensify** [comment]
@aimane-chnaif , once we've settled on the deliverable for this issue (and we can create other issues for updates, as needed), can you please be sure to start reviewing proposals from the top > down?

**Comment by ericsonrd** [comment]
> > About this, do you think it's better to use themeColors.textSupporting or themeColors.iconHovered?
> 
> Ah, if we already have iconHovered, let's just go with that.
> 
> > made the test and 4px adds a lot of spacing for this component (in terms of communicating that this icon belongs to the email address preceeding it). I made an example with 4px and 2px (which I'd propose) so you can see the difference:
> 
> I think 4px looks cleaner, and then we need to make sure there is 4px of margin to the right of the icon as well.

Ok. Here's a test with 4px left and right on Web and Native:

![expMarginleftright](https://user-images.githubusercontent.com/95394815/232572423-36cfbf82-a85a-4c23-860c-8a32d6ca818d.jpg)

**Comment by shawnborton** [comment]
Nice, I think the right version looks better but what exactly does remove string spacing mean?

**Comment by ericsonrd** [comment]
> Nice, I think the right version looks better but what exactly does remove string spacing mean?

This one in en.js (also in the spanish version): 
<img width="785" alt="emptyStringSpacing" src="https://user-images.githubusercontent.com/95394815/232628672-ad81ab75-9392-44c9-ab5f-8e2b48f18e7e.png">

**Comment by shawnborton** [comment]
Ah, what if we just don't put any right margin on the icon, or only put 2px right margin on it?

**Comment by ericsonrd** [comment]
> Ah, what if we just don't put any right margin on the icon, or only put 2px right margin on it?

With no margin to the right, seems like the same result but with the benefit of not having to modify `en.js`. Here are the tests:

![expMarginleft](https://user-images.githubusercontent.com/95394815/232636119-49ab37c9-5e04-479d-8778-379796912b6e.jpg)

**Comment by flaviadefaria** [comment]
Just to confirm, is this [issue](https://github.com/Expensify/App/issues/17481) also being fixed here?

**Comment by flaviadefaria** [comment]
@robertjchen can you please confirm so that I can potentially close the other GH? Thanks!

**Comment by robertjchen** [comment]
@flaviadefaria Yep, let's close the other one. This one will address it 👍

**Comment by robertjchen** [comment]
Updated the `Expected Result` in the OP issue as well. It seems like we're waiting on some final design input https://github.com/Expensify/App/issues/16574#issuecomment-1512260463 and we can move forward with a proposal.

**Comment by dukenv0307** [comment]
@aimane-chnaif My solution is also simple by use `Pressable` wrap `Icon` same as `BaseMiniContextMenuItem`. You can try it here: https://github.com/dukenv0307/App/tree/fix/17362

**Comment by dukenv0307** [comment]
@aimane-chnaif Did you mean my root cause doesn't explain space logic?

**Comment by dukenv0307** [comment]
@aimane-chnaif I just update my [proposal](https://github.com/Expensify/App/issues/16574#issuecomment-1511137193) and branch

**Comment by Natnael-Guchima** [comment]
Bringing this [issue](https://github.com/Expensify/App/issues/17751) here, if the issue is needed to be addressed here

**Comment by MelvinBot** [comment]
@shawnborton @robertjchen @mallenexpensify @aimane-chnaif this issue is now 4 weeks old and preventing us from maintaining WAQ, can you:
- Decide whether any proposals currently meet our guidelines and can be approved as-is today
- If no proposals meet that standard, please take this issue internal and treat it as one of your highest priorities
- If you have any questions, don't hesitate to start a discussion in #expensify-open-source

Thanks!

**Comment by MelvinBot** [comment]
Current assignee @aimane-chnaif is eligible for the Internal assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Current assignee @mallenexpensify is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Current assignee @aimane-chnaif is eligible for the External assigner, not assigning anyone new.

**Comment by mallenexpensify** [comment]
@situchan and @aimane-chnaif hired in Upwork, please accept 
https://www.upwork.com/jobs/~0125054b3431e347fc

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.3.6-0 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/17949


If no regressions arise, payment will be issued on 2023-05-04. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@aimane-chnaif] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@aimane-chnaif] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@aimane-chnaif] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@aimane-chnaif] Determine if we should create a regression test for this bug.
- [ ] [@aimane-chnaif] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@mallenexpensify] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by Natnael-Guchima** [comment]
Hello @mallenexpensify. I have reported similar issue on Contact methods page [here ](https://github.com/Expensify/App/issues/17362), and it was decided to be addressed with a single PR  on this issue and the issue that I reported was closed. Is this something I am eligible for bonus reporting or is this something that is considered as a duplicate? 
Thanks.

**Comment by mallenexpensify** [comment]
Posted on the other issue @Natnael-Guchima 
https://github.com/Expensify/App/issues/17362#issuecomment-1529961573

**Comment by Natnael-Guchima** [comment]
> Posted on the other issue @Natnael-Guchima 
> https://github.com/Expensify/App/issues/17362#issuecomment-1529961573
> 
> 

Thanks for the reply. I just read that now.

**Comment by mallenexpensify** [comment]
(sorry, doubling up to get clarification) 

@Natnael-Guchima it appears that your report is after the others and they're pretty similar
<img width="625" alt="image" src="https://user-images.githubusercontent.com/22508304/235493296-8daa2eb0-ecf8-4a33-a720-e11e52bc94a0.png">

<img width="623" alt="image" src="https://user-images.githubusercontent.com/22508304/235493454-df2f0877-c36d-42fb-b0d2-20f0e8b0a0bd.png">

The deciding factor might be, would this instance have been fixed if you hadn't reported it?  Or... did this instance get added to the other issue/PR ?   If it's the former, I don't think you'd be eligible for the reporting bonus.  If the latter, it makes sense to me that you would (especially since we'd rather have it _lumped_ in with the other bug instead of have this separate issue we need to complete)

@aimane-chnaif for the above, do you think the former or the latter is true?

**Comment by dhanashree-sawant** [comment]
Hi @mallenexpensify, ping for payment.

**Comment by robertjchen** [comment]
not overdue, pending payment

**Comment by mallenexpensify** [comment]
Thanks for the ping, apologies for the delay. 
@situchan and @aimane-chnaif paid $3000 each which includes the timeliness bonus. 
@dhanashree-sawant can you please accept the job and reply here once you have? 
https://www.upwork.com/jobs/~0125054b3431e347fc

**Comment by dhanashree-sawant** [comment]
Hi  @mallenexpensify, thanks I have accepted the offer.

**Comment by mallenexpensify** [comment]
@dhanashree-sawant paid $250 for reporting, thanks! 

Added to the Design Guidelines section of the Testing Guidelines docs [here](https://docs.google.com/document/d/1UaWlcieiwLnsKfeKw7iwLF7vgYRmdw_-K8OcY5aPBRU/edit#heading=h.r44oknvea8fx)
